integration/ide/bsp-server/src/BspServerTests.scala

```diff
@@ -15,6 +15,26 @@ object BspServerTests extends UtestIntegrationTestSuite {
   override protected def workspaceSourcePath: os.Path =
     super.workspaceSourcePath / "project"
 
+  def substitutions(
```

Comment:
Let's call this `substituteTransitiveDependencies` or something, the current name is a bit too concise given it's not obvious what it's trying to do